etc/adminhtml/system/adyen_pay_by_mail.xml

```diff
@@ -47,18 +47,23 @@
             <tooltip>How many days do you want this payment page to be active</tooltip>
             <config_path>payment/adyen_pay_by_mail/session_validity</config_path>
         </field>
-        <field id="skin_code" translate="label" type="text" sortOrder="50" showInDefault="1" showInWebsite="1" showInStore="1">
+        <field id="ship_before" translate="label" type="text" sortOrder="50" showInDefault="1" showInWebsite="1" showInStore="1">
+            <label>Ship Before Date</label>
+            <tooltip>The date within which the ordered goods or services need to be shipped or provided to the shopper</tooltip>
```

Comment:
I see this config is type "text" but the tooltip says it is a date, the expected value is the number of days right? In that case I would change the label to "Ship Before Date (Days)" to keep the same name as in the docs. The Tooltip can also say something like "Days after the the Pay by Mail is generated within which the ordered goods or services..."

Also, let's add a number validation like in line 42 of this file

Reply:
Added the validator to ship_before and also session_validity, too.